src/components/StudentProfileEdit/EditProfile.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import axios from "axios";
+import "./EditProfile.css";
+
+class EditProfile extends Component {
+  /*   state = {
+    email: this.props.email
+  }; */
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "sadsadsa",
+      country: "egypt",
+      phone: "61111",
+      birth_date: "date",
+      password: "464468",
```

Comment:
Take the initial values through props, 
It will be like this 
``` email: this.props.email ```

Reply:
**_ok mr.HazemGabr232_**